zulipterminal/config/keys.py

```diff
@@ -400,6 +405,11 @@ class KeyBinding(TypedDict):
         'help_text': 'Swap with previous character',
         'key_category': 'editor_text_manipulation',
     },
+    'NEW_LINE': {
+        'keys': ['enter'],
+        'help_text': 'Insert new line',
+        'key_category': 'msg_compose',
+    },
```

Comment:
In some ways this comes across like one of those obvious ones like `backspace`, though I agree that it could depend on 'enter to send' - so this is fine for now, and balances out the explicit calling out of `enter` for other purposes elsewhere in the UI.

However, we may want to add a comment here about those points, or possibly connect it to the urwid-readline 'suffix' work, since that is where the multi-line key comes from.

Reply:
Thank you for the pointer!
I've added a comment.
Let me know if it needs to be edited.